### DISCUSSION TITLE
TypeError: argument of type 'int' is not iterable

### DISCUSSION THREAD

**Discussion by AyhemAllouache** [ppogg/YOLOv5-Lite#235] [Q&A]
(yolo-env) pi@raspberrypi:~/Desktop/YOLOv5-Lite $ python3 detect.py --source 0
Namespace(weights='weights/v5lite-e.pt', source='0', img_size=640, conf_thres=0.45, iou_thres=0.5, device='', view_img=False, save_txt=False, save_conf=False, nosave=False, classes=None, agnostic_nms=False, augment=False, update=False, project='runs/detect', name='exp', exist_ok=False)
YOLOv5 🚀 v1.4-53-g0f96139 torch 2.0.1 CPU

Fusing layers... 
/home/pi/Desktop/yolo-env/lib/python3.9/site-packages/torch/functional.py:504: UserWarning: torch.meshgrid: in an upcoming release, it will be required to pass the indexing argument. (Triggered internally at /root/pytorch/aten/src/ATen/native/TensorShape.cpp:3483.)
  return _VF.meshgrid(tensors, **kwargs)  # type: ignore[attr-defined]
Model Summary: 167 layers, 781205 parameters, 0 gradients, 2.9 GFLOPS
qt.qpa.xcb: QXcbConnection: XCB error: 148 (Unknown), sequence: 185, resource id: 0, major code: 140 (Unknown), minor code: 20
1/1: 0... Traceback (most recent call last):
  File "/home/pi/Desktop/YOLOv5-Lite/detect.py", line 178, in <module>
    detect()
  File "/home/pi/Desktop/YOLOv5-Lite/detect.py", line 51, in detect
    dataset = LoadStreams(source, img_size=imgsz, stride=stride)
  File "/home/pi/Desktop/YOLOv5-Lite/utils/datasets.py", line 279, in __init__
    if 'youtube.com/' in url or 'youtu.be/' in url:  # if source is YouTube video
TypeError: argument of type 'int' is not iterable

**Comment by meimingziqule** [comment]
解决了吗老哥  我也遇到这问题了

**Reply by meimingziqule** [reply]
[YOLOv5-Lite-master.zip](https://github.com/user-attachments/files/16933235/YOLOv5-Lite-master.zip)
遇到这个问题的哥们可以用用这个代码  直接运行就可以调用摄像头了  转自https://blog.csdn.net/m0_46935103/article/details/134946848?ops_request_misc=%257B%2522request%255Fid%2522%253A%2522BE82802D-7094-438C-AB6D-B1C9F5293732%2522%252C%2522scm%2522%253A%252220140713.130102334.pc%255Fall.%2522%257D&request_id=BE82802D-7094-438C-AB6D-B1C9F5293732&biz_id=0&utm_medium=distribute.pc_search_result.none-task-blog-2~all~first_rank_ecpm_v1~rank_v31_ecpm-5-134946848-null-null.142^v100^pc_search_result_base2&utm_term=yolov5-lite%E6%97%A0%E6%B3%95%E8%B0%83%E7%94%A8%E6%91%84%E5%83%8F%E5%A4%B4&spm=1018.2226.3001.4187
